kernel_config

```diff
@@ -4201,26 +3201,13 @@ CONFIG_CRYPTO_DEV_PADLOCK_SHA=y
 # CONFIG_CRYPTO_DEV_CCP is not set
 # CONFIG_ASYMMETRIC_KEY_TYPE is not set
 CONFIG_HAVE_KVM=y
-CONFIG_HAVE_KVM_IRQCHIP=y
-CONFIG_HAVE_KVM_IRQ_ROUTING=y
-CONFIG_HAVE_KVM_EVENTFD=y
-CONFIG_KVM_APIC_ARCHITECTURE=y
-CONFIG_KVM_MMIO=y
-CONFIG_KVM_ASYNC_PF=y
-CONFIG_HAVE_KVM_MSI=y
-CONFIG_HAVE_KVM_CPU_RELAX_INTERCEPT=y
-CONFIG_KVM_VFIO=y
 CONFIG_VIRTUALIZATION=y
-CONFIG_KVM=m
-CONFIG_KVM_INTEL=m
-CONFIG_KVM_AMD=m
-CONFIG_KVM_DEVICE_ASSIGNMENT=y
+# CONFIG_KVM is not set
```

Comment:
I'm on the fence on this one.  It's not terribly useful inside a VM, but in a boot2docker-on-bare-metal world it's awesome to have `/dev/kvm` available.  Maybe keep this one as a module for now? :angel: